storage-s3-dynamodb/src/main/java/io/delta/storage/BaseExternalLogStore.java

```diff
@@ -188,9 +231,11 @@ protected void writeActions(
         Path path,
         Iterator<String> actions
     ) throws IOException {
-        LOG.debug("writeActions to: {}", path);
+        if (LOG.isDebugEnabled()) {
```

Comment:
We don't need to check this. The `debug` method we call below will check it for you.

Here is an example when we should check the flag:
```
if (LOG.isDebugEnabled()) {
   String path = // some heavy work
   LOG.debug("writeActions to: {}", path);
}
```

Same for other log changes.

Reply:
sounds good.